pym/bob/cmds/jenkins.py

```diff
@@ -848,6 +848,11 @@ def ensureFingerprint(step):
                         AUDIT=JenkinsJob._auditName(d),
                         WSP_PATH=d.getWorkspacePath()))
 
+        # remove @tmp directory generated by some jenkins plugins
+        for d in sorted(self.__checkoutSteps.values()):
+            prepareCmds.append("# remove @tmp directories created by some jenkins plugins")
+            prepareCmds.append("rm -rf {}".format(" ".join(d.getWorkspacePath() + "/" + s + "@tmp" for s in d.getScmDirectories())))
```

Comment:
Better quote the argument to `rm -rf`...

```suggestion
            prepareCmds.append("rm -rf {}".format(" ".join(quote(d.getWorkspacePath() + "/" + s + "@tmp") for s in d.getScmDirectories())))
```